resources/eventing/charts/nats/README.md

```diff
@@ -1,33 +1,102 @@
-# NATS Chart
+# NATS Server
+
+This Helm chart deploy NATS: https://nats.io/ using NATS [Helm chart](https://github.com/nats-io/k8s/tree/v0.9.0/helm/charts/nats).
 
-This Helm chart deploy NATS: https://nats.io/  using NATS-Operator.
 
 Steps:
 
 - Install NATS into "nats" namespace using Helm 3 :
 ```bash
-$ helm install nats nats -n nats --set install.enabled=true --create-namespace --debug --wait
+kubectl create namespace nats
+helm template nats nats -n nats | kubectl apply -f -
 ```
 - Test the installation:
 ```bash
 $ kubectl -n nats port-forward nats-1 4222
 ```
-Open two terminals.
-In the first one, create "foo" subject and subscribe to it using client ID 1:
-```bash
-$ telnet localhost 4222
-SUB foo 1
+
+## Configuration
+
+### Limits
+
+```yaml
+nats:
+  # The number of connect attempts against discovered routes.
+  connectRetries: 30
+
+  # How many seconds should pass before sending a PING
+  # to a client that has no activity.
+  pingInterval: 
+
+  # Server settings.
+  limits:
+    maxConnections: 
+    maxSubscriptions: 
+    maxControlLine: 
+    maxPayload: 
+
+    writeDeadline: 
+    maxPending: 
+    maxPings: 
+    lameDuckDuration: 
+
+  # Number of seconds to wait for client connections to end after the pod termination is requested
+  terminationGracePeriodSeconds: 60
 ```
-In the second terminal, publish a message to the "foo" subject. The lenght of the message must be snet before the content
-```bash
-$ telnet localhost 4222
-PUB foo 10
-Hello Kyma
-PUB foo 3
-Bye
+
+### Logging
+
+*Note*: It is not recommended to enable trace or debug in production since enabling it will significantly degrade performance.
```

Comment:
```suggestion
> **NOTE**: It is not recommended to enable trace or debug in production, since enabling it will significantly degrade performance.
```